README.md

```diff
@@ -1,5 +1,53 @@
-# Memgraph Query Modules
+# MAGE - Memgraph Advanced Graph Extensions :crystal_ball:
 
-This repository is one place for all Memgraph query modules. You can find and
-contribute implementations of various algorithms in various programming
-languages, all runnable inside Memgraph.
+This open-source repository contains all available query modules written by the team behind Memgraph and its users. You can find and contribute implementations of various algorithms in multiple programming languages, all runnable inside Memgraph. This project aims to give everyone the tools they need to tackle the most challenging graph problems. 
+
+## MAGE Query modules
+
+Memgraph introduces the concept of **query modules**, user-defined procedures that extend the Cypher query language. These procedures are grouped into modules that can be loaded into Memgraph. You can find more information on query modules in the official [documentation](https://docs.memgraph.com/memgraph/database-functionalities/query-modules).
+
+Query modules implemented in Python:
+* [wcc.py](python/wcc.py): The WCC (Weakly Connected Components) query module can run WCC analysis on a sub-graph of the whole graph.
```

Comment:
Maybe create a wiki here on github with all the algos?

Reply:
The Wiki section is enabled :smile: